usr/sbin/so-clear-backlog

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+. /usr/sbin/so-common
+
+if [ -d /nsm/sensor_data ]; then
+	:
+else
+	echo "Not a sensor!  Exiting..."
+	exit 1;
+fi
+
+SKIP=0
+
+#########################################
+# Options
+#########################################
+usage()
+{
+cat <<EOF
+Security Onion Backlog Purge
+  Options:
+  -h         This message
+  -y         Skip interactive mode
+
+EOF
+}
+
+while getopts "h:y" OPTION
+do
+        case $OPTION in
+                h)
+                        usage
+                        exit 0
+                        ;;
+                y)
+                        SKIP=1
+                        ;;
+                *)
+                        usage
+                        exit 0
+                        ;;
+        esac
+done
+
+ENGINE=$(grep ENGINE /etc/nsm/securityonion.conf | cut -d= -f2)
```

Comment:
This can have unexpected behavior if there are multiple matches for ENGINE.  

Can we just `source /etc/nsm/securityonion.conf`?

Reply:
That sounds better.